### PR TITLE
adding endpoint for associated engagements

### PR DESCRIPTION
**Pull request by jayprakash1**

### CODE REVIEW

File: lib/engagement.js [hunk 1]
```diff
@@ -16,6 +16,19 @@ class Engagement {
     }, cb)
   }
 
+  associatedEngagements (objectType, objectId, options, cb) {
```
Comment (pcothenet) [inline lib/engagement.js]: Hi @jayprakash1, thanks for the contribution.

This doesn't quite match the naming convention for methods. I'm not familiar with this particular API call, but what about `getAssociated`?

**Comment by pcothenet** [comment]
If you can also add it to the Readme, that would be perfect!

**Comment by jayprakash1** [comment]
Please check now.

**Comment by pcothenet** [comment]
Awesome!

**Pull request merged**